### DISCUSSION TITLE
I got error status code 500 when I run command "nuxt dev"

### DISCUSSION THREAD

**Discussion by Kriz0911** [vuejs/vuefire#1468] [Questions and Help]
I setting nuxt-vuefire in nuxt.config
then run command nuxt dev
compile is success 

but the window show error status code 500
[vite-node] [ERR_LOAD_URL] @posva/lru-cache
at @posva/lru-cache

p.s nuxt generate is success work

<img width="1132" alt="截圖 2023-11-28 上午9 36 03" src="https://github.com/vuejs/vuefire/assets/31040024/79d1673e-94f2-4fff-a773-c815cfa7c411">
<img width="488" alt="截圖 2023-11-28 上午9 36 22" src="https://github.com/vuejs/vuefire/assets/31040024/d18fabb1-6446-4663-af3e-01a799fc4a4a">

package
    "nuxt": "^3.8.2",
    "vue": "^3.3.8",
    "firebase": "^10.6.0",
    "nuxt-vuefire": "^0.4.1",
    "vuefire": "^3.1.18"

Please tell me how can I fix it, thanks.

**Comment by posva** [comment]
I don't know where the problem is coming from but you should be able to get started from one of these starters: https://vuefire.vuejs.org/nuxt/deployment.html#Deployment
